### DISCUSSION TITLE
Change Vercel serverless function memory

### DISCUSSION THREAD

**Discussion by liegeandlief** [sveltejs/kit#8812] [Q&A] [answered]
Is it possible to change the memory allocated to the serverless functions when running on Vercel?

They seem to use the default of 1024mb and I'd like to reduce this as my functions don't need that much.

I tried adding the following to my project's vercel.json file:

```
	"functions": {
		"**/*": {
			"memory": 256
		}
	}
```

But this gave the error `The pattern "**/*" defined in `functions` doesn't match any Serverless Functions inside the `api` directory.` when running the `vercel build` command.

**Comment by Rich-Harris** [comment, marked as answer]
This will be made possible by https://github.com/sveltejs/kit/pull/8740
